package.json

```diff
@@ -24,16 +24,44 @@
     "mozvr"
   ],
   "scripts": {
-    "start": "webpack-dev-server --config ./config/webpack-dev.js --progress --colors",
-    "build": "webpack --config ./config/webpack-dev.js --mode production",
-    "publish": "git subtree push --prefix examples github/digaverse gh-pages"
+    "start": "webpack-dev-server --config ./config/webpack-dev.js --mode development --progress --colors",
+    "build": "webpack --config ./config/webpack-prod.js --mode production",
+    "publish": "git subtree push --prefix examples github/digaverse gh-pages",
+    "lint": "./node_modules/.bin/eslint tests/ src/ config/",
+    "test": "karma start ./tests/karma.conf.js",
+    "test:firefox": "karma start ./tests/karma.conf.js --browsers Firefox",
+    "test:chrome": "karma start ./tests/karma.conf.js --browsers Chrome"
   },
   "dependencies": {
     "resonance-audio": "^1.0.0"
   },
   "devDependencies": {
+    "aframe": "^0.8.2",
+    "browserify": "^16.2.2",
+    "chai": "^4.1.2",
+    "chai-shallow-deep-equal": "^1.4.6",
     "clean-webpack-plugin": "^0.1.19",
+    "eslint": "^5.1.0",
+    "eslint-config-standard": "^11.0.0",
+    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-node": "^6.0.1",
+    "eslint-plugin-promise": "^3.8.0",
+    "eslint-plugin-standard": "^3.1.0",
+    "karma": "^2.0.4",
+    "karma-browserify": "^5.3.0",
+    "karma-chai-shallow-deep-equal": "0.0.4",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-env-preprocessor": "^0.1.1",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-sinon-chai": "^1.3.4",
+    "log-suppress": "^1.0.3",
```

Comment:
Shouldn't use `log-suppress` since it is stale repository and requires `"engines": { "node": "5.0.0" }`

Reply:
Okay, I've manually disabled console.log. Else the console is cluttered with irrelevant information during testing.